src/modules/navigator/rtl_params.c

```diff
@@ -103,3 +103,14 @@ PARAM_DEFINE_FLOAT(RTL_LAND_DELAY, -1.0f);
  * @group Return To Land
  */
 PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
+
+/**
+ * Climb to highest altitude flown for RTL
+ *
+ * This enables the vehicle to climb to the highest altitude flown since last reset
```

Comment:
consider clarifying this comment to say something like: "When engaging RTL, this enables the vehicle to climb to whichever is higher: RTL_RETURN_ALT or the highest altitude flown since last reset"